docker/.docker-env

```diff
@@ -0,0 +1,5 @@
+SECRET_WORD=AJSD-s*a3w-a2wd-Sk0)-ls91k-JW2*@~
+PORT=3000
+NODE_ENV=dev
+RSASigningSchema=pkcs1-sha256
+AdminPublicKey=-----BEGIN PUBLIC KEY----- MIGfMA0GCSqGSIb3DQEBAQUAA4GNADCBiQKBgQDlOJu6TyygqxfWT7eLtGDwajtN FOb9I5XRb6khyfD1Yt3YiCgQWMNW649887VGJiGr/L5i2osbl8C9+WJTeucF+S76 xFxdU6jE0NQ+Z+zEdhUTooNRaY5nZiu5PgDB0ED/ZKBUSLKL7eibMxZtMlUDHjm4 gwQco1KRMDSmXSMkDwIDAQAB -----END PUBLIC KEY-----

```

Comment:
I know this got merged already, but in a future merge/update, this file should get removed. These env variables are no longer used throughout the project.